From 96576124252305d23e486e068d8abe0b0cc1bc93 Mon Sep 17 00:00:00 2001
From: Alex Doyle <adoyle@nvidia.com>
Date: Fri, 18 Dec 2020 10:16:28 -0800
Subject: [PATCH] brace-initialization-patch for pesign-113

Fix strict warning message treated as an error
(-Werror=missing-braces) by explicitly initializing every struct
member to zero.
---
 libdpe/pe_allocspace.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libdpe/pe_allocspace.c b/libdpe/pe_allocspace.c
index a2898f6..cdd6a5d 100644
--- a/libdpe/pe_allocspace.c
+++ b/libdpe/pe_allocspace.c
@@ -61,7 +61,7 @@ pe_set_image_size(Pe *pe)
 	struct pe32plus_opt_hdr *opthdr = pe->state.pe32plus_exe.opthdr;
 
 	Pe_Scn *scn = NULL;
-	struct section_header shdr = { "", 0, }, tmp_shdr;
+	struct section_header shdr = {{ 0 }, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, tmp_shdr;
 	if (pehdr->sections < 1)
 		return -1;
 
-- 
2.20.1

